src/form/AsyncSelect/AsyncSelectStyles.ts

```diff
@@ -0,0 +1,139 @@
+import { StylesConfig } from 'react-select';
+import { themeColor as themeColorAsc, themeSpacing as themeSpacingAsc, calculateFluidStyle } from 'asc-ui-beta';
+import styled from 'styled-components';
+import { Props } from './AsyncSelect';
+import { ascDefaultTheme } from 'asc-ui-beta';
+
+// Proxy to add the used theme
+function themeColor(type: string, variant?: string) {
```

Comment:
No need for @ts-ignore here. Please ask the team before resorting to these, thanks :pray: 

```typescript
// Proxy to add the used theme
function themeColor(type?: Parameters<typeof themeColorAsc>[0], variant?: Parameters<typeof themeColorAsc>[1]) {
	return themeColorAsc(type, variant)({ theme: ascDefaultTheme });
}
// Proxy to add the used theme
function themeSpacing(top: number, right?: number, bottom?: number, left?: number) {
	return themeSpacingAsc(top, right, bottom, left)({ theme: ascDefaultTheme });
}
```

This only leaves me with:
`TS2345: Argument of type '"level"' is not assignable to parameter of type 'ColorType | undefined'.`
From here:
```typescript
singleValue: (styles) => ({
	...styles,
	color: `${themeColor('level', 'level6')}`,
}),
```

Reply:
Do you why that error would show up?

Reply:
you are asking me about TS, right :D

Reply:
fixed